Help/FAQ.rst

```diff
@@ -93,3 +93,71 @@ our master branch. Right now we use 7 of those checks:
 
 - **continuous-integration/appveyor/pr** This one does the same as the one
   above, but for Windows, both 32 and 64bits versions.
+
+How are labels used in issues and pull requests?
+------------------------------------------------
+
+``area`` labels refer to a particular section in the code base.
+
+- ``area/actions`` 
+- ``area/bears`` tools to implement bears
+- ``area/CI`` continuous integration
+- ``area/CLI``
+- ``area/core`` coala core
+- ``area/documentation`` docstrings, readthedocs
+- ``area/results``
+- ``area/tests``
+- ``area/upstream``
+- ``area/usability``
+- **coala-bears** specific: ``area/lintbears`` 
+
+``difficulty`` labels indicate that an issue is ready to work on. 
+
+- ``difficulty/newcomer`` typos and straightforward fixes to help new 
+  contributors get acquainted with the workflow
```

Comment:
we should mention that people shouldn't fix more than one newcomer issue if it's not an urgent or important one, rather file those